ros_buildfarm/templates/snippet/publisher_xunit.xml.em

```diff
@@ -13,6 +13,7 @@ assert type_tag in ('CTestType', 'GoogleTestType', 'JUnitType'), 'Unsupported te
 }@
         <@(type_tag)>
           <pattern>@ESCAPE(pattern)</pattern>
+          <excludesPattern/>
```

Comment:
* xunit report types can now take an exclude pattern as well as the
include pattern. At one point we split up some test type patterns to
deconflict between multiple test types but I'm not sure that a simple
exclude would allow us to re-combine any of them I will have to check
the history.